src/components/ProgressForm/ProgressHeader.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from '@/components/ProgressForm/ProgressForm.module.scss';
+
+function ProgressHeader() {
+    return (
+        <>
+            <header className={styles.header}>
+                <div>
+                <p>You have <span className={styles.mark}> 2 missed </span> Progress Updates</p>
+                <p>Lets try to avoid missing updates</p>
+                </div>
```

Comment:
why is this extra uneccesary div added here?

Reply:
Missed on that one